app/assets/javascripts/code/services/google/graph_highlight.js

```diff
@@ -15,7 +15,11 @@ angular.module("google").factory("graphHighlight",  ["map",
       this.hide();
       var self = this;
       _(points).each(function(point){
-        self.items.push({marker: map.drawMarker(point), point: point});
+        self.items.push({marker: map.drawMarker({
```

Comment:
Have you considered extracting something here? It seems a bit noisy

Reply:
Yes! But I don't want to move too many stuff at once or the whole messed up ecosystem might collapse. I'm leaving graph_highlight.js and note.js for the next round :)